tdrs-backend/tdpservice/users/api/authorization_check.py

```diff
@@ -20,7 +20,14 @@ def get(self, request, *args, **kwargs):
         """Handle get request and authenticate user."""
         user = request.user
         if user.is_authenticated:
-            auth_params = {"authenticated": True, "user": {"email": user.username}}
+            auth_params = {
+                "authenticated": True,
+                "user": {
```

Comment:
I wonder if here you'd be able to take advantage of the [UserSerializer](https://github.com/HHS/TANF-app/pull/55/files#diff-eb126d5c965960aa16268e1579183160R8) and nest it: https://www.django-rest-framework.org/api-guide/serializers/#dealing-with-nested-objects

Reply:
As discussed during the [09/15/2020 dev pairing session](https://hhsgov.sharepoint.com/sites/TANFDataPortalOFA/_layouts/15/Doc.aspx?OR=teams&action=edit&sourcedoc={22F8187F-7AC2-4198-8673-7C111DDFD4E5})  this was a suggestion. Due to the simple nature of the JSON returned in this response the overhead of implementing the UserSerializer and nested object support would be unnecessary unless the request grows to be more complex.